buildtools/build.proj

```diff
@@ -52,84 +52,95 @@
     <Target Name="test-blueprints-dotnew" DependsOnTargets="package-blueprints;build-nuget-packages">
         <RemoveDir Directories="../Deployment/BlueprintGenerationTests/DotnetNew"/>
         <MakeDir Directories="../Deployment/BlueprintGenerationTests/DotnetNew"/>
-        <MakeDir Directories="../Deployment/BlueprintGenerationTests/DotnetNew/netcore21"/>
-        <MakeDir Directories="../Deployment/BlueprintGenerationTests/DotnetNew/netcore31"/>
+        <MakeDir Directories="../Deployment/BlueprintGenerationTests/DotnetNew/vs2022"/>
```

Comment:
do we need anything that works with vs2017 anymore in this Target?

Reply:
My thought process was I wasn't going to remove the v2017 blueprints while we still have VS 2017 support just in case there was some really weird issue but since 2.1 is out of support from both MS and Lambda I didn't think it was worth being part of the build. If we had a need to make a 2017 change we could add it back.